### DISCUSSION TITLE
Group Assignment #4

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#135] [Assignments]
# Survey Diagram and Website Wire-frame
## Due 5/6

Have a group member comment on this following thread with the following:
- Link or photos for the survey diagram 
- Link or photos for the website wire-frame.

Submit your revised survey using this form:
https://forms.gle/xNH4fTWGP4ZQ6VjR7

**Comment by ghost** [comment]
[Reimagining 6th Wireframe.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6431766/Reimagining.6th.Wireframe.pdf)
[Diagram.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6431769/Diagram.pdf)

**Reply by albertkun** [reply]
@ktgreenler Wow, I REALLY like the wireframe and diagram!!! Great job!!! One question, I see it says 1/2, is there a second page?

**Comment by rachan2023** [comment]
[Against Asian Hate Diagram.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6432069/Against.Asian.Hate.Diagram.pdf)

[AsianAmericanHate_Wireframe.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6432036/AsianAmericanHate_Wireframe.pdf)

**Reply by albertkun** [reply]
@rachan2023 I like the diagram! For the wireframe, is the "stopaapi hate" logo a placeholder? Hopefully so, as keeping it there means there's some affiliation to the project and yours!

**Reply by avaitt** [reply]
@albertkun ya it's just a placeholder. I'll make sure to remove it in the final project!

**Comment by jason-muljadi** [comment]
[SEA Admit Weekend Wireframe.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6432654/SEA.Admit.Weekend.Wireframe.pdf)
[SEA Admit Weekend Diagram.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6432655/SEA.Admit.Weekend.Diagram.pdf)

**Comment by nnhimichelle** [comment]
[surveydiagram.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6436337/surveydiagram.pdf)
[wireframe.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6436340/wireframe.pdf)

**Comment by alizezila** [comment]
Migrant Justice Group (Anny, Andi, Alize) 
[Migrant Justice Wireframe.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6436697/Migrant.Justice.Wireframe.pdf)
[Migrant Justice Diagram.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6436698/Migrant.Justice.Diagram.pdf)

**Comment by rh5140** [comment]
LA Tenants Group (Sam @skarakas1, Brian @brillianz, Ray @rh5140)
[Link to wireframe](https://drive.google.com/file/d/113rakppIQy89k3iLUZ6ga_jigMRXUFNV/view?usp=sharing)
[Link to survey flow diagram](https://drive.google.com/file/d/1ls3TfahQO3LF97hC7M1rc91ndCDQDBRk/view?usp=sharing)

**Comment by Wny-Duong** [comment]
Health Equity Group (Latisha, Ericko, Wilny)
[Here's the wireframe.](https://docs.google.com/document/d/1_-dVmGmLwGe0EI01wIylhyrkS63WNqrHCuxVIP8Jdt8/edit)
[Here's the survey diagram](https://drive.google.com/file/d/16XI-icSPyV9tC6MXbvrbF40PXvOmgRuc/view?usp=sharing)
